CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 Magento Functional Testing Framework Changelog
 ================================================
+
+2.6.0
+-----
+
+* Traceability
+    * MFTF generated cest files are fully compatible for Codeception `dry-run`.
+* Modularity
+    * `mftf generate:tests` and `mftf run:test` commands now accept suite scoped test names in format `[<suitename:>testname]...`.
+* Maintainability
+    * Support `deprecated` syntax for the following test entities:
+        * Test
+        * Action Group
+        * Data
+        * Page
+        * Section
+        * Section Element
+        * Suite
+        
```

Comment:
No empty line, include line 21 in the same bulleted list.

Reply:
Changed